### PR TITLE
Supported basic auth on download action

### PR DESCRIPTION
**Pull request by thangnc**
**CHANGES**
- Supported basic auth on download action

### CODE REVIEW

**Review by Martin-Gonzalez90 — CHANGES_REQUESTED** [review]

**Comment by Martin-Gonzalez90** [comment]
> **CHANGES**
> 
> * Supported basic auth on download action

Thanks a lot for contributing to the package! There are some changes to make before releasing the new version. :)

**Comment by thangnc** [comment]
Sorry, because i thought you are not active on this repo. Then i cloned and renamed plugin to support basic auth for my company's project. But you asked to update the PR, then I will clone and re-create PR to keep this plugin

**Comment by MartinGonzalez** [comment]
> Sorry, because i thought you are not active on this repo. Then i cloned and renamed plugin to support basic auth for my company's project. But you asked to update the PR, then I will clone and re-create PR to keep this plugin

No problem at all. If you can create a PR to add these changes to this repo would be amazing to keep growing this plugin :)

File: README.md [hunk 1]
```diff
@@ -17,7 +17,8 @@
 
 ## Getting Started
 
-This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To get started with `fastlane-plugin-json`, add it to your project by running:
+This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To get started with `fastlane-plugin-json`,
```
[the remark this answers is not in the document]
Comment (thangnc) [reply README.md:20]: Sr, i forked this repo and make the new plugin to fix my company's project ASAP. Thats why i need to update this file

**Pull request closed without merging**